### PR TITLE
New development url changes

### PR DESCRIPTION
**Pull request by philwiffen**
includes migration guide, notice, and updates to other docs that mention the old repo

### CODE REVIEW

File: projects/tech-zone/new-development-url-migration-guide.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+To do this you’ll need to:
+
+1.  Clone the new repository in GitHub Desktop, which will get the new location on your local computer
+1.  Remove the old repository from your local file system – so that you don’t accidentally develop in there once migrated.
+
+## Clone the new repository in Github desktop
+
+1.  Open GitHub Desktop
+1.  Either press Ctrl+T to bring up your repositories, or click on the "Current repository" drop down in the UI
```
Comment (martinzugec-ctx) [inline projects/tech-zone/new-development-url-migration-guide.md]: Either use mouse (image with highlighted control to select), or include macOS keyboard shortcuts.

UPDATE: Just finished reading the second half of the sentence :) I would just switch the order (mouse first as it's applicable to everyone).

File: projects/tech-zone/new-development-url-notice.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+This will give Citrix a central place for contributors to submit to, that doesn’t have an individual owner.
+
+## What exactly is changing, and what isn't?
+
+Tech Zone is moving from contributors creating content in a Github organisation owned by an individual (martinzugec-ctx), to a Github organisation owned by the Tech Zone team (citrixtechzone).
+
+This will mean that you will need to change where you clone from and push to. But it doesn't change the general contribution process.
+
+## What do I need to do?
```
Comment (martinzugec-ctx) [inline projects/tech-zone/new-development-url-notice.md:28]: "What if I have an active branch in the old environment"?

**Review by martinzugec-ctx — APPROVED** [review]
Looks good, only minor suggestions

File: projects/tech-zone/new-development-url-notice.md (hunk 2 above)
Comment (philwiffen) [reply projects/tech-zone/new-development-url-notice.md:28]: Good point! I've added some details

File: projects/tech-zone/new-development-url-migration-guide.md (hunk 1 above)
Comment (philwiffen) [reply projects/tech-zone/new-development-url-migration-guide.md]: I made it mouse-only :)

**Pull request merged**